### ISSUE TITLE
Broken Aphorism Tile Texture

### ISSUE THREAD

**Issue by MasterEnderman** [TeamPneumatic/pnc-repressurized#209]
### Minecraft Version
1.12.2
### Forge Version
2729
### Mod Version
0.7.0-230 (jenkins)
### Describe your problem, including steps to reproduce it
![image](https://user-images.githubusercontent.com/24524662/42447754-34f69d1c-837b-11e8-8d48-efc4b752aa91.png)

**Comment by MasterEnderman** [comment]
It's a custom one without any support for pneumaticcraft.

**Comment by desht** [comment]
Strange.  I did change the Aphorism tile model in 0.7.0, to support recolouring.  However, it's working fine here:

![aphorism](https://user-images.githubusercontent.com/736831/42448628-b23513ea-8375-11e8-9157-413e4fb6a7ce.png)

**Comment by desht** [comment]
Think I might need to see your (full) startup log, to see if there are any mode loading errors.

**Comment by MasterEnderman** [comment]
I looked through my logs and i think that is the part you're interested in:
```
[15:42:30] [main/ERROR] [FML.TEXTURE_ERRORS]: ==================================================
[15:42:30] [main/ERROR] [FML.TEXTURE_ERRORS]:   DOMAIN pneumaticcraft
[15:42:30] [main/ERROR] [FML.TEXTURE_ERRORS]: --------------------------------------------------
[15:42:30] [main/ERROR] [FML.TEXTURE_ERRORS]:   domain pneumaticcraft is missing 1 texture
[15:42:30] [main/ERROR] [FML.TEXTURE_ERRORS]:     domain pneumaticcraft has 1 location:
[15:42:30] [main/ERROR] [FML.TEXTURE_ERRORS]:       mod pneumaticcraft resources at G:\Games\Minecraft\MultiMC\instances\GrischaLabs\.minecraft\mods\pneumaticcraft-repressurized-1.12.2-0.7.0-230.jar
[15:42:30] [main/ERROR] [FML.TEXTURE_ERRORS]: -------------------------
[15:42:30] [main/ERROR] [FML.TEXTURE_ERRORS]:     The missing resources for domain pneumaticcraft are:
[15:42:30] [main/ERROR] [FML.TEXTURE_ERRORS]:       textures/blocks/aphorism_tile_border.png
[15:42:30] [main/ERROR] [FML.TEXTURE_ERRORS]: -------------------------
[15:42:30] [main/ERROR] [FML.TEXTURE_ERRORS]:     No other errors exist for domain pneumaticcraft
[15:42:30] [main/ERROR] [FML.TEXTURE_ERRORS]: ==================================================
```

**Comment by desht** [comment]
Huh.  The version I uploaded to Curseforge indeed missing that (new) texture. 

OK, guess it's time for 0.7.2... :)

**Comment by desht** [comment]
Build 232 should fix that problem.  I'm not going to do a 0.7.2 release just yet in case any other teething problems crop up :)

But 0.7.2 within a day or two.

**Comment by desht** [comment]
Fixed in 0.7.2 release
